src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -388,6 +403,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
                     me={this.props.me}
                     onSelect={value => onSelectSavedAddress(value)}
                     handleClickEdit={this.handleClickEdit}
+                    inCollectorProfile={false}
```

Comment:
We could make this prop optional and not have to pass it where its not needed, but its seems safer to have this additional check here.